core/sql/parser/sqlparser.y

```diff
@@ -24909,7 +24946,6 @@ create_table_as_token: TOK_AS
 
 /* type pElemDDL */
 table_definition_body : table_element_list
-                      | like_definition
                       | external_table_definition
 
```

Comment:
It looks like the new like option is no longer part of a table_definition_body but as a separate high-level production.  Would this break some operations?  For example, I see on line 24594 that a "like" option is syntactically correct with <special_table_name> option.  Would this still be allowed with your change?